### ISSUE TITLE
Please explicitly link against -lpthread

### ISSUE THREAD

**Issue by apoleon** [stransky/berusky2#23]
Hello,

I'm filing a new bug report because this is not a Debian specific issue and also affects Fedora and other distributions.

It seems you use an older version of Fedora where ld allows the user to link indirectly against needed libraries. This behaviour is changing now:

https://fedoraproject.org/wiki/UnderstandingDSOLinkChange

You can also learn more about this issue here:

https://wiki.debian.org/ToolChain/DSOLinking#Not_resolving_symbols_in_indirect_dependent_shared_libraries

So in short -lpthread needs to be linked in explictly in the linker stage. I'm attaching a patch that solves the problem.

https://gist.github.com/apoleon/7305027
